RDF.js

```diff
@@ -749,14 +749,20 @@ function importItem(newItem, node) {
 		var creatorType = possibleCreatorTypes[i];
 		if(creatorType == "author") {
 			creators = getFirstResults(node, [n.bib+"authors", n.dc+"creator", n.dc1_0+"creator",
-				n.dcterms+"creator", n.eprints+"creators_name",
+				n.dcterms+"creator", n.eprints+"creators_name", n.article+"author", n.video+"director", n.music+"musician",
```

Comment:
hmm -- I hadn't seen the bit about the Facebook profile URLs. The idea to include this originally came from here:
https://hbr.org/2015/08/how-to-do-walking-meetings-right
which just has regular names (we also pick the authors from the byline, but I figured that wouldn't always be the case.).

So given that, my suggestion would be to take them out here and put them in the EM translator and don't use the content where it contains facebook.com -- does that sound good?